.github/workflows/test-internet.yml

```diff
@@ -7,17 +7,15 @@ on:
 
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
-    paths:
-      - test/internet/**
+    paths: [test/internet/**]
```

Comment:
How does this improve the consistency? Are we following a rule or use an auto-formatting tool to decide what syntax to use?

Reply:
I noticed in the different files that when the list is short, this syntax is used.
Unfortunately, Yamllint does not have a rule to enforce this choice.

Reply:
Not blocking comment: I don't think we should use this format of list here, if we add more paths in the future, the other syntax will be more convceniant.

Reply:
If we add more paths in the future and the list becomes more difficult to read or a bit long, we can always use the other syntax.